src/Our.Umbraco.Ditto.Contrib/Collections/ChildrenAs.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+
+namespace Our.Umbraco.Ditto
+{
+    public class ChildrenAs : DittoMultiProcessorAttribute
+    {
+        public ChildrenAs(string documentTypeAlias = null)
```

Comment:
Wonder if we should have a constructorless version which just does [UmbracoProperty("Children")] as a convenience thing?